api/managementpb/actions.proto

```diff
@@ -47,8 +47,7 @@ message StartMySQLExplainActionRequest {
   string pmm_agent_id = 1;
   // Service ID for this Action. Required.
   string service_id = 2 [(validator.field) = {string_not_empty: true}];
-  // Query ID or query is required.
-  // Value of query.
+  // Deprecated: should not be used.
```

Comment:
let's also add that it will be removed in 2.42

Reply:
and in other places too.

Reply:
Done.